packages/cli/src/utils/templates.ts

```diff
@@ -41,10 +41,15 @@ module.exports = {
           from: { height: "var(--radix-accordion-content-height)" },
           to: { height: "0" },
         },
+        spinner: {
+          from: { opacity: '1' },
+          to: { opacity: '0.15' },
+        },
       },
       animation: {
         "accordion-down": "accordion-down 0.2s ease-out",
         "accordion-up": "accordion-up 0.2s ease-out",
+        spinner: 'spinner 1.2s linear infinite',
```

Comment:
I think this animation could be renames to something like `fade-out` since its only using opacity and doesnt describe any spinning animation.  Will also make it more composable with other components 🤔

Reply:
thinking for a sec, something like this could work: `spinner: 'rotate 1.2s infinite steps(12)',` and then you can just stagger the opacity and rotate the icon 🤷‍♂️